tools/lxdclient/client.go

```diff
@@ -288,11 +325,10 @@ func checkLXDBridgeConfiguration(conf string) error {
 			}
 		} else if strings.HasPrefix(line, "LXD_BRIDGE=") {
 			name := strings.Trim(line[len("LXD_BRIDGE="):], " \"")
-			/* If we're here, we want lxdbr0 to be configured
-			 * because the default profile that juju will use says
-			 * lxdbr0. So let's fail if it doesn't.
-			 */
-			if name != "lxdbr0" {
+			// If we're here, we want the bridge to be configured because the
+			// default profile that juju will use says lxdbr0. So let's fail if
```

Comment:
Actually, no - this specific helper func is only called to verify the default lxdbr0 config is sane. If the user changed the default profile's bridge name, it won't be ever called, so the comment still applies (will revert the formatting change though).